### DISCUSSION TITLE
Will Android-Password-Store support GnuPG's new OCB (AEAD) block cipher?

### DISCUSSION THREAD

**Discussion by l3u** [zeapo/Android-Password-Store#2936] [General]
Hi all,

GnuPG 2.4 is becoming stable in more and more distributions. Newly generated keys will set the controversial OCB (AEAD) block cipher as their default. This leads to incompatibilities with other PGP implementations, above all OpenKeychain (cf. https://github.com/open-keychain/open-keychain/issues/2900 and more).

Now it seems that Android-Password-Store is being ported away from OpenKeychain, and will use PGPainless.

So: Will this support the unchanged new GnuPG keys as-is? Or will we have the same compatibility problems as with OpenKeychain right now?

Thanks for all clarification!

Cheers, Tobias

**Comment by msfjarvis** [comment]
PGPainless is not planning to support the GnuPG AEAD extension so you'll see the same issues.

**Comment by l3u** [comment]
Thanks for the info!
